sotodlib/toast/ops/sim_sso.py

```diff
@@ -224,21 +224,30 @@ def _exec(self, data, detectors=None, **kwargs):
 
         return
 
-    def _get_planet_temp(self, sso_name):
+    @function_timer
+    def _get_sso_temperature(self, sso_name):
         """
-        Get the thermodynamic planet temperature given
+        Get the thermodynamic SSO temperature given
         the frequency
         """
+        log = Logger.get()
         dir_path = os.path.dirname(os.path.realpath(__file__))
         hf = h5py.File(os.path.join(dir_path, "data/planet_data.h5"), "r")
-        if sso_name in hf.keys():
+        if sso_name == "Moon":
+            freq = np.linspace(0, 1000, 1001)[1:] * u.GHz
+            T = 300 # Kelvin
+            emissivity = 1.0
+            tb = 1 / (k / (h * freq) * np.log(1 + (np.exp(h * freq / (k * T)) - 1) / emissivity))
```

Comment:
What is happening with this line?  Are k & h defined in this scope?  Looks like a stray...

Reply:
That is an incomplete attempt to add a simple emission model of the Moon. I just pushed the completed version. Here is the same simulation as above but now using the emission model:

![image](https://user-images.githubusercontent.com/596250/215157114-8af17d4b-7d4f-4503-a2a2-94237ae52092.png)

Reply:
@Zhilei that cooler band going through the source is indeed from filtering.